content/pt-br/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Agentes de execução do Contêiner
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+{{< note >}}
+Dockershim foi removido do projeto Kubernetes desde a versão 1.24. Para mais detalhes leia [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq).
+{{< /note >}}
+
+Você precisa instalar um {{< glossary_tooltip text="agente de execução do contêiner" length="all"  term_id="container-runtime" >}} em cada nó do cluster para que os pods possam ser executados. Esta página descreve as tarefas e o que está envolvido para configurar os nós. 
+
+O Kubernetes 1.24 requer um tempo de execução que esteja em conformidade com {{< glossary_tooltip term_id="cri" length="all" text="Container Runtime Interface">}} (CRI).
+
+Para mais informações, veja [versão de suporte CRI](#cri-versions).
+
+Esta página fornece uma descrição de como utilizar múltiplos container runtime comuns com Kubernetes.
```

Comment:
container runtime needs to be translated

Reply:
Yes, I forgot that one

Reply:
Let's use "agentes de execução de contêiner" instead of "container runtime".